### PR TITLE
Core/PacketIO: Updated SMSG_QUESTGIVER_STATUS_MULTIPLE and SMSG_QUEST…

### PR DESCRIPTION
**Pull request by Faq**
Updated SMSG_QUESTGIVER_STATUS_MULTIPLE and SMSG_QUESTGIVER_STATUS

**Changes proposed:**

-  SMSG_QUESTGIVER_STATUS_MULTIPLE to SMSG_QUEST_GIVER_STATUS_MULTIPLE
-  SMSG_QUESTGIVER_STATUS to SMSG_QUEST_GIVER_STATUS
-  update packet structure

**Target branch(es):** 3.3.5/master

- [x] 3.3.5
- [ ] master

**Tests performed:**
Compiles

**Known issues and TODO list:** (add/remove lines as needed)

- [ ] Test ingame

### CODE REVIEW

**Comment by jackpoz** [comment]
I would like to see https://github.com/TrinityCore/TrinityCore/issues/25313 fixed before continuing with changing more opcodes

**Comment by offl** [comment]
https://github.com/TrinityCore/TrinityCore/issues/25331 too and is more critical

**Comment by jackpoz** [comment]
Due to the recent issues caused by packet classes ports that added little value and broke ingame features, we are pausing accepting PR's that attempt to port packets to the new classes, especially if they are not small packets with 1 field that makes it very very easy to spot issues.

Let's just have 2 weeks in a row where all packets just work.

**Comment by Faq** [comment]
Then why close this? keep it like draft for 2w...

**Comment by jackpoz** [comment]
Let me rephrase that:
Let's just never have a broken opcode in 3.3.5 branch again
